src/migration/1723579275223-SourceCategory.ts

```diff
@@ -0,0 +1,40 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceCategory1723579275223 implements MigrationInterface {
+  name = 'SourceCategory1723579275223';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "source_category" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "title" text NOT NULL, "enabled" boolean NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "UQ_66d6dccf282b8104ef9c44c0fb5" UNIQUE ("title"), CONSTRAINT "PK_21e4d5359f2a23fd10053f516e9" PRIMARY KEY ("id"))`,
+    );
+    await queryRunner.query(`ALTER TABLE "source" ADD "categoryId" uuid`);
+    await queryRunner.query(
+      `ALTER TABLE "source" ADD CONSTRAINT "FK_02e1cbb6e33fa90e68dd56de2a9" FOREIGN KEY ("categoryId") REFERENCES "source_category"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`,
+    );
+    await queryRunner.query(`
+      INSERT INTO "source_category"
```

Comment:
Can we maybe also add a seed for local just now we working on it?

Reply:
Is this not the same thing? Locally, you will have to run migrations, and this will fill up your DB.

Reply:
I can turn this into a seed, if that is more ideal. Wdyt?

Reply:
Oh sorry I kind of added the one before realizing this.
This will do :)